src/views/blog-post/series/series-toc.module.scss

```
@@ -1,63 +1,168 @@
+	--series-nav_header_title_color: var(--foreground_emphasis-high);
+
+	--series-nav_chapter-container_color: var(--surface_primary_emphasis-none);
+
+	--series-nav_chapter-item_border_color_hovered: var(--primary_variant);
+	--series-nav_chapter-item_border_color_pressed: var(--primary_default);
+	--series-nav_chapter-item_border_color_selected: var(
+		--surface_primary_emphasis-medium
+	);
+
+	--series-nav_chapter-item_background-color_hovered: var(
+		--surface_primary_emphasis-none
+	);
+	--series-nav_chapter-item_background-color_pressed: var(
+		--surface_primary_emphasis-low
+	);
+	--series-nav_chapter-item_arrow_color: var(--foreground_emphasis-medium);
+	--series-nav_chapter-item_title_color: var(--primary_default);
+	--series-nav_chapter-item_title_color_pressed: var(
+		--foreground_emphasis-high
+	);
+	--series-nav_chapter-item_title_color_selected: var(
+		--foreground_emphasis-medium
+	);
+}
+
 .seriesTableOfContent {
-	margin-bottom: 16px;
-
-	@include from($desktopSmall) {
-		margin-bottom: 24px;
-	}
-
-	.seriesHeader {
-		color: var(--darkPrimary);
-		margin-bottom: 16px;
-		margin-top: 0;
-	}
-
-	ol {
-		padding: 0;
-		list-style: none;
-		margin: 0;
-	}
-
-	li {
-		position: relative;
-		padding-left: 40px;
-	}
-
-	li {
-		margin-top: 16px;
-	}
-
-	li > button {
-		background: none;
-		border: none;
-		color: var(--darkPrimary);
-		font-size: inherit;
-		text-decoration: underline;
-		cursor: pointer;
-	}
-
-	li::before {
-		position: absolute;
-		height: 8px;
-		width: 8px;
-		left: 8px;
-		top: 50%;
-		transform: translateY(-50%);
-		border-radius: 50%;
-		content: " ";
-		box-sizing: border-box;
-	}
-
-	li:not(.isActive)::before {
-		border: 1.2px solid var(--darkPrimary);
-	}
-
-	li.isActive::before {
-		background: var(--darkPrimary);
-	}
-
-	.displayNone {
-		display: none;
-	}
+	padding: var(--series-nav_padding-vertical) 0;
+	max-width: var(--max-width_s);
+}
+
+.titleContainer {
+	margin: 0 !important;
+}
+
+.partOfSeries {
+	font-style: italic;
+	display: block;
+	margin: 0;
+	color: var(--series-nav_header_overline_color);
+	margin-bottom: var(--series-nav_header_overline_padding-bottom);
+}
+
+.articleTitle {
+	color: var(--series-nav_header_title_color);
+	margin: 0;
+}
+
+.seriesHeader {
+	padding: 0 var(--series-nav_header_padding-horizontal)
+		var(--series-nav_header_padding-bottom);
+}
+
+.contentsContainer {
+	background-color: var(--series-nav_chapter-container_color);
+	border-radius: var(--series-nav_chapter-container_corner-radius);
+	padding: var(--spc-4x) 0 0;
+}
+
+.listContainer {
+	list-style: none;
+	margin: 0;
+	padding: 0;
+}
+
+.navigationItemOuter[data-dont-show-initially] {
+	display: none;
+}
+
+.navigationItem {
+	display: block;
+	position: relative;
+	padding: var(--series-nav_chapter-item_padding-vertical)
+		var(--series-nav_chapter-item_padding-horizontal);
+	margin: 0;
+	text-decoration: none;
+	color: var(--series-nav_chapter-item_title_color);
```

```suggestion
	color: var(--series-nav_chapter-item_title_color);
	@include transition(color background-color);
```

Also, transitions for hover/focus states... :)